src/electron/README.md

```diff
@@ -27,28 +14,21 @@ To run the Electron clients, run:
 npm run action electron/start [windows|linux]
 ```
 
-## Windows
+## Cross-Compiling
 
-To build for Windows on a macOS, you need to first install [MinGW-w64](https://www.mingw-w64.org/) v11.0.1+.
+To build the app for a platform target on a different host target, you will need a cross-compiler. We use [zig to cross-compile with cgo](https://dev.to/kristoff/zig-makes-go-cross-compilation-just-work-29ho).
 
-With [MacPorts](https://www.mingw-w64.org/downloads/#macports) (official channel):
+[Install zig](https://ziglang.org/learn/getting-started/#installing-zig) and make sure it's in the PATH.
 
-```sh
-sudo port install x86_64-w64-mingw32-gcc @11.0.1
-```
-
-With Homebrew (unofficial, how to ensure consistent version?):
+You can download the binary tarball, or use a package manager, like Homebrew:
```

Comment:
A link to the supported package managers may be useful. It covers almost all platforms including Windows, macOS and Ubuntu.

```suggestion
You can download the binary tarball, or use a [package manager](https://github.com/ziglang/zig/wiki/Install-Zig-from-a-Package-Manager), like Homebrew:
```